step1__select_samples.py

```diff
@@ -83,12 +77,9 @@ def main(args):
     )
     mt = mt.select_entries("GT", "GQ", "DP")
 
-    logger.info("Filtering to entries meeting GQ and DP thresholds")
-    mt = mt.filter_entries(
-        (mt.GQ >= gq_threshold)
-        & (mt.DP >= dp_threshold_expr(mt, dp_threshold))
-        & (mt.GT.is_non_ref())
-    )
+    logger.info("Filtering to entries meeting GQ, DP and other 'adj' thresholds")
+    mt = annotate_adj(mt)
```

Comment:
You can also use `filter_to_adj` if you want instead

Reply:
Thanks - switched to that.